Samples/Sample XML/StudentTranscript.xml

```diff
@@ -518797,6 +518797,7 @@
 				<Term>uri://ed-fi.org/TermDescriptor#Fall Semester</Term>
 			</StudentAcademicRecordIdentity>
 		</StudentAcademicRecordReference>
+		<ExternalEducationOrganizationNameOfInstitution>Kahn Academy</ExternalEducationOrganizationNameOfInstitution>
```

Comment:
Was this spelling intentional or just a typo?  I'm assuming these were meant to reference Khan Academy?

Reply:
ah typo. thank you for the catch! i'll put in that fix.